components/TimeoutModal.tsx

```diff
@@ -55,7 +55,12 @@ export const TimeoutModal: React.FC<TimeoutModalProps> = ({ timedOut, userArrive
     // And at the end, send back to EDD
     setTimeout(() => {
       if (typeof window !== 'undefined') {
-        const eddLoginLink = getUrl('edd-log-in')?.concat(encodeURIComponent(window.location.toString()))
+        // Note that the concatenated portion of this link is functionally useless, as IDM is not currently
+        // able to redirect based on the resource_url parameter concatenated.
```

Comment:
Why are we including the concatenated portion of the link if it's functionally useless?

Reply:
Because we didn't know it was functionally useless at the time, I imagine. I don't want to risk taking it out in this PR, so I added a comment instead.

Reply:
👍 Got it. Sounds good